lib/commands/gesture.js

```diff
@@ -142,15 +148,25 @@ helpers.handleScroll = async function (gestures) {
 };
 
 helpers.handleDrag = async function (gestures) {
-  // get gestures
-  let press = gestures[0];
-  let wait = gestures[1];
-  let moveTo = gestures[2];
-
-  // get drag data
-  let pressCoordinates = await this.getCoordinates(press);
-  let duration = (parseInt(wait.options.ms, 10) / 1000);
-  let moveToCoordinates = await this.getCoordinates(moveTo);
+  let pressCoordinates;
+  let duration;
+  let moveToCoordinates;
+  if (gestures.length === 4) {
+    let press = gestures[0];
```

Comment:
can we use [fail-sort destruction](http://es6-features.org/#FailSoftDestructuring) here?